content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -1,162 +1,144 @@
+of the control plane.
 {{< /note >}}
 
-The original plan to support volumes using CCM was to use [Flex](/docs/concepts/storage/volumes/#flexVolume) volumes to support pluggable volumes. However, a competing effort known as [CSI](/docs/concepts/storage/volumes/#csi) is being planned to replace Flex.
-
-Considering these dynamics, we decided to have an intermediate stop gap measure until CSI becomes ready.
-
-## Functions of the CCM
-
-The CCM inherits its functions from components of Kubernetes that are dependent on a cloud provider. This section is structured based on those components.
-
-### 1. Kubernetes controller manager
-
-The majority of the CCM's functions are derived from the KCM. As mentioned in the previous section, the CCM runs the following control loops:
-
-* Node controller
-* Route controller
-* Service controller
+## Cloud controller manager functions {#functions-of-the-ccm}
 
-#### Node controller
+The controllers inside the cloud controller manager include:
 
-The Node controller is responsible for initializing a node by obtaining information about the nodes running in the cluster from the cloud provider. The node controller performs the following functions:
+### Node controller
 
-1. Initialize a node with cloud specific zone/region labels.
-2. Initialize a node with cloud specific instance details, for example, type and size.
-3. Obtain the node's network addresses and hostname.
-4. In case a node becomes unresponsive, check the cloud to see if the node has been deleted from the cloud.
-If the node has been deleted from the cloud, delete the Kubernetes Node object.
+The node controller is responsible for creating {{< glossary_tooltip text="Node" term_id="node" >}} objects
+when new servers are created in your cloud infrastructure. The node controller obtains information about the
+hosts running inside your tenancy with the cloud provider. The node controller performs the following functions:
 
-#### Route controller
+1. Initialize a Node object for each server that the controller discovers via the cloud provider API.
+2. Annotating and labelling the Node object with cloud-specific information, such as the region the node
+   is deployed into and the resources (CPU, memory, etc) that it has available.
+3. Obtain the node's hostname and network addresses.
+4. Verifying the node's health. In case a node becomes unresponsive, this controller checks with
+   your cloud provider's API to see if the server has been deactivated / deleted / terminated.
+   If the node has been deleted from the cloud, the controller deletes the Node object from your Kubernetes
+   cluster.
 
-The Route controller is responsible for configuring routes in the cloud appropriately so that containers on different nodes in the Kubernetes cluster can communicate with each other. The route controller is only applicable for Google Compute Engine clusters.
+Some cloud provider implementations split this into a node controller and a separate node
```

Comment:
nit: Some cloud provider implementations split this?

Reply:
Not sure what you're saying here @kbhawkey. Are you suggesting a revision to:
```suggestion
Some cloud provider implementations split this.
```
?

Reply:
@sftim, Reading the sentence, what is `this` referring to?
Some cloud providers divide [split | separate] the implementation of the node controller between a node controller and a node lifecycle controller